ci-scripts/setup-cluster.sh

```diff
@@ -123,6 +127,21 @@ EOF
     wait_for_entity_by_selector 300 openshift-pipelines pod app=tekton-pipelines-webhook
     kubectl -n openshift-pipelines wait --for=condition=ready --timeout=300s pod -l app=tekton-pipelines-webhook
 
+    info "Enable performance options"
+    chains_perf_options=""
+    if [ -n "$chains_kube_api_qps" ] || [ -n "$chains_kube_api_burst" ] || [ -n "$chains_threads_per_controller" ]; then
+        if [ -n "$chains_kube_api_qps" ]; then
+            chains_perf_options+="\"--kube-api-qps=$chains_kube_api_qps\","
+        fi
+        if [ -n "$chains_kube_api_burst" ]; then
+            chains_perf_options+="\"--kube-api-burst=$chains_kube_api_burst\","
+        fi
+        if [ -n "$chains_threads_per_controller" ]; then
+            chains_perf_options+="\"--threads-per-controller=$chains_threads_per_controller\""
```

Comment:
If `$chains_threads_per_controller` would not be set, you would get coma at the end of the `$chains_perf_options` which would be invalid JSON.